Makefile.include

```diff
@@ -7,6 +7,7 @@ RIOTBASE ?= $(MCUFIRMWAREBASE)/../RIOT/
 RIOTPKG ?= $(MCUFIRMWAREBASE)/pkg
 EXTERNAL_BOARD_DIRS ?= $(MCUFIRMWAREBASE)/boards/
 EXTERNAL_MODULE_DIRS += $(MCUFIRMWAREBASE)/drivers/lds01
+EXTERNAL_MODULE_DIRS += $(MCUFIRMWAREBASE)/drivers/lds01_dma
```

Comment:
EXTERNAL_MODULE_DIRS will be set using MCUFW_MODULE_DIRS below.

You can remove this line.

Reply:
Done